atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceIntegrationTest.java

```diff
@@ -300,6 +301,54 @@ public void deletionTakesPlaceAtFreshTimestamp() throws Exception {
         assertThat(results).doesNotContainKey(CELL);
     }
 
+    @Test
+    public void testGetRowsForDeletion() throws Exception {
```

Comment:
the deletion at this point isn't really important, in your test you have no deletion code anyway (not that I'm asking you to do so either), should name this along the lines of "testGetRowsDoesNotReadOldData" or something along those lines